src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/RazorLSPTextViewConnectionListener.cs

```diff
@@ -239,7 +239,7 @@ private static void InitializeRazorTextViewOptions(IVsTextManager4 textManager,
             var wordWrapStyle = WordWrapStyles.None;
             if (Convert.ToBoolean(langPrefs3[0].fWordWrap))
             {
-                wordWrapStyle |= WordWrapStyles.WordWrap;
+                wordWrapStyle |= WordWrapStyles.WordWrap | WordWrapStyles.AutoIndent;
```

Comment:
Oooooh shit, had no clue. Out of curiosity how did you end up finding this bit? From my understanding tracking these types of bugs down is a royal pain 😄